src/python/pants/backend/python/rules/importable_python_sources.py

```diff
@@ -1,38 +1,59 @@
 # Copyright 2020 Pants project contributors (see CONTRIBUTORS.md).
 # Licensed under the Apache License, Version 2.0 (see LICENSE).
-
 from dataclasses import dataclass
+from typing import Tuple
 
 from pants.backend.python.rules.inject_init import InitInjectedSnapshot, InjectInitRequest
 from pants.backend.python.rules.inject_init import rules as inject_init_rules
 from pants.backend.python.target_types import PythonSources
 from pants.core.target_types import FilesSources, ResourcesSources
 from pants.core.util_rules import determine_source_files
 from pants.core.util_rules.determine_source_files import AllSourceFilesRequest, SourceFiles
+from pants.core.util_rules.strip_source_roots import representative_path_from_address
 from pants.engine.fs import Snapshot
 from pants.engine.rules import RootRule, rule
-from pants.engine.selectors import Get
+from pants.engine.selectors import Get, MultiGet
 from pants.engine.target import Sources, Targets
+from pants.source.source_root import SourceRoot, SourceRootRequest
 
 
 @dataclass(frozen=True)
 class ImportablePythonSources:
```

Comment:
The names in this file might not make sense anymore. How about the file is `python_sources.py` and we have `StrippedPythonSources` and `UnstrippedPythonSources`?